config/foreman.migrations/20200305172758_enable_puma.rb

```diff
@@ -0,0 +1 @@
+answers['foreman']['passenger'] = false if answers['foreman'].is_a?(Hash)
```

Comment:
If you add `&& answers['foreman']['passenger']` at the end it makes the tests happy

Reply:
Thank you for the suggestion